www/documentation/getting_started/run_on_android.html

```diff
@@ -279,6 +279,7 @@
 <pre><code>&gt; python make_apk.py --package=org.crosswalkproject.example \
     --manifest=xwalk-simple/manifest.json</code></pre>
 <p>This will package the application defined in the specified <code>manifest.json</code> file and produce two apk files from it, one for x86 architecture and one for ARM. The apk files will end up in the directory where you ran the script. Each file is given the name set in the manifest, with any filesystem-sensitive characters removed and an architecture identifier (&quot;x86&quot; or &quot;arm&quot;) appended. For our example, the output files are <code>simple_x86.apk</code> and <code>simple_arm.apk</code>.</p>
+<p>Please specify --app-versionCode or --app-versionCodeBase, otherwise android:versionCode will be generated based on android:versionName. For example, versionName "3.2.1" will get versionCode: ABI prefix + "0302001".</p>
 </li>
```

Comment:
I just mean that we say “use –app-versionCode or –app-versionCodeBase”, but in the previous code example right before that paragraph it does not use either of those parameters:

> python make_apk.py --package=org.crosswalkproject.example \
    --manifest=xwalk-simple/manifest.json

Should we add –app-versionCode=”0.0.1” ?
Bob

From: Han Leon [mailto:notifications@github.com]
Sent: Tuesday, January 27, 2015 1:16 AM
To: crosswalk-project/crosswalk-website
Cc: Spencer, Bob
Subject: Re: [crosswalk-website] add some explanation about android:versionCode generation (#194)


In www/documentation/getting_started/run_on_android.html<https://github.com/crosswalk-project/crosswalk-website/pull/194#discussion_r23592457>:

> @@ -279,6 +279,7 @@

>  <pre><code>&gt; python make_apk.py --package=org.crosswalkproject.example \

>      --manifest=xwalk-simple/manifest.json</code></pre>

>  <p>This will package the application defined in the specified <code>manifest.json</code> file and produce two apk files from it, one for x86 architecture and one for ARM. The apk files will end up in the directory where you ran the script. Each file is given the name set in the manifest, with any filesystem-sensitive characters removed and an architecture identifier (&quot;x86&quot; or &quot;arm&quot;) appended. For our example, the output files are <code>simple_x86.apk</code> and <code>simple_arm.apk</code>.</p>

> +<p>Please specify --app-versionCode or --app-versionCodeBase, otherwise android:versionCode will be generated based on android:versionName. For example, versionName "3.2.1" will get versionCode: ABI prefix + "0302001".</p>

>  </li>

Thanks a lot for information and kindly help.

And, I'm not quite sure what should I do about the following comment. Could you explain more? Thanks.
"it might be good to demonstrate what we are asking in the previous code block."

—
Reply to this email directly or view it on GitHub<https://github.com/crosswalk-project/crosswalk-website/pull/194/files#r23592457>.